zipline/data/minute_bars.py

```diff
@@ -809,11 +832,12 @@ def _get_metadata(self):
 
     @lazyval
     def last_available_dt(self):
-        return self._market_closes[-1]
+        _, close = self.calendar.open_and_close_for_session(self._end_session)
+        return close
 
     @property
     def first_trading_day(self):
```

Comment:
We could push this further and also rename the property, but that has more touch points, so may be better for a different PR.